src/pages/JavaThreadDumpAnalyzer.jsx

```
@@ -0,0 +1,246 @@
+import React, { useState, useCallback } from 'react';
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)